absl/debugging/symbolize_win32.inc

```diff
@@ -45,13 +45,26 @@ void InitializeSymbolizer(const char*) {
   // symbols be loaded. This is the fastest, most efficient way to use
   // the symbol handler.
   SymSetOptions(SYMOPT_DEFERRED_LOADS | SYMOPT_UNDNAME);
-  if (!SymInitialize(process, nullptr, true)) {
-    // GetLastError() returns a Win32 DWORD, but we assign to
-    // unsigned long long to simplify the ABSL_RAW_LOG case below.  The uniform
-    // initialization guarantees this is not a narrowing conversion.
-    const unsigned long long error{GetLastError()};  // NOLINT(runtime/int)
-    ABSL_RAW_LOG(FATAL, "SymInitialize() failed: %llu", error);
+  DWORD syminitialize_error;
+  constexpr int kSymInitializeRetries = 5;
+  for(int i = 0; i < kSymInitializeRetries; i++) {
+    if (SymInitialize(process, nullptr, true)) {
+      return;
+    }
+
+    // SymInitialize can fail sometimes with STATUS_INFO_LENGTH_MISMATCH
+    // (0xC0000004). In this case, we should try calling SymInitialize again.
+    syminitialize_error = GetLastError();
+    if (syminitialize_error != 0xC0000004) {
```

Comment:
Should we use the STATUS_INFO_LENGTH_MISMATCH constant?  

The "windows.h" header include above should define this header through "ntstatus.h".

Reply:
Using the constant now. Thanks!